meta-mender-demo/recipes-mender/mender-client/mender-client-append.inc

```diff
@@ -1,14 +1,14 @@
-MENDER_UPDATE_POLL_INTERVAL_SECONDS:mender-client-install = "5"
-MENDER_INVENTORY_POLL_INTERVAL_SECONDS:mender-client-install = "5"
-MENDER_RETRY_POLL_INTERVAL_SECONDS:mender-client-install = "30"
+MENDER_UPDATE_POLL_INTERVAL_SECONDS:mender-update-install = "5"
+MENDER_INVENTORY_POLL_INTERVAL_SECONDS:mender-update-install = "5"
+MENDER_RETRY_POLL_INTERVAL_SECONDS:mender-update-install = "30"
 
-MENDER_UPDATE_CONTROL_MAP_EXPIRATION_TIME_SECONDS:mender-client-install = "90"
-MENDER_UPDATE_CONTROL_MAP_BOOT_EXPIRATION_TIME_SECONDS:mender-client-install = "45"
+MENDER_UPDATE_CONTROL_MAP_EXPIRATION_TIME_SECONDS:mender-update-install = "90"
```

Comment:
Might as well remove these or?

Reply:
They are still applicable to the old client versions. This file should not be be picked up by the C++ client, since it has the `mender-client` name in it.